Tom/Classes/CornerRadius.swift

```diff
@@ -7,7 +7,10 @@
 
 import Foundation
 
+/// Keys that specify a corner radius type.
 enum CornerRadius {
+    /// Keys that specify corner radius half of view's height.
     case rounded
+    /// Keys that specify corner radius for developer selected radius
     case custom(CGFloat)
 }
```

Comment:
I think no need any comment for this class because nobody will able to see it from the property summary